### PR TITLE
MEPTS-1510 | Lista de Crianças Inscritas em CCR NEW REPORT

### PR DESCRIPTION
**Pull request by helderjosue**

### CODE REVIEW

**Review by TinoMuianga — COMMENTED** [review]
The code looks good overall, but there are conflicts that need to be resolved before it can pass for a second review and be merged. Please resolve the conflicts and let me know so I can review it again.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ccr/ListOfChildrenEnrolledInCCRDataDefinitionQueries.java
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ccr/ListOfChildrenEnrolledInCCRDataDefinitionQueries.java]: This inner Join is not necessary, you could just **Select from () ccr_nid** and join with the second part to avoid unnecessary complexity.
Also remove the **comma** after pi.identifier column returned on ccr_nid to avoid breaking the code

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ccr/ListOfChildrenEnrolledInCCRDataDefinitionQueries.java
Comment (helderjosue) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ccr/ListOfChildrenEnrolledInCCRDataDefinitionQueries.java]: This code will only select patients with ccr_nid marked as **preferred**, what happens if the patient also has NID Tarv? Make sure you also include patients with multiple NIDs when they are enrolled in more than one program on the system.
Confirm with the BA team

**Review by psmatsinhe — APPROVED** [review]
Looks good to me

**Pull request merged**
